src/conv/invokers/impl_gemm_dynamic.cpp

```diff
@@ -107,6 +229,39 @@ InvokerFactory MakeImplGemmDynamicDataInvokerFactory(const ConvolutionContext& c
             };
         };
     }
+    else if(ctx.direction.IsBackwardData())
+    {
+        return [ctx](const std::vector<Kernel>& kernels) {
+            return [=](const Handle& handle, const boost::any& primitive_parameters) {
+                const auto data_ctx = boost::any_cast<conv::DataInvokeParams>(primitive_parameters);
+                const auto& tensors = data_ctx.tensors;
+                auto kernel         = handle.Run(kernels[0]);
+                if(kernel.GetName().find("igemm_bwd_gtc") == 0)
+                {
+                    std::vector<KernelInvoke> ks;
+                    std::transform(kernels.begin(),
+                                   kernels.end(),
+                                   std::back_inserter(ks),
+                                   [&](const Kernel& k) { return handle.Run(k); });
+                    float elapsed = 0;
+
+                    elapsed = CallImplicitGemmDynamic(
+                        handle, ctx, tensors.in, tensors.out, tensors.w, ks);
+
+                    if(handle.IsProfilingEnabled())
+                    {
+                        handle.ResetKernelTime();
+                        handle.AccumKernelTime(elapsed);
+                    }
+                }
+                else
+                {
+                    MIOPEN_THROW(
+                        "Error running dynamic implicit GEMM convolution (invalid kernel name?)");
+                }
+            };
```

Comment:
Please remove this no-op semicolon after } 
One more down below,

Reply:
Do you mean this `MIOPEN_THROW` to be put into another place outside the inner most lambda?
Not sure if that is OK, but currently the logic is the same as that in hip implicit gemm. https://github.com/ROCmSoftwarePlatform/MIOpen/blob/develop/src/conv/invokers/impl_gemm.cpp#L171

Reply:
I am talking about semicolon here after curly braces ```};```

Reply:
Oh you are right. Let me remove it